packages/hurlfmt/src/curl/matches.rs

```diff
@@ -77,6 +77,15 @@ pub fn headers(arg_matches: &ArgMatches) -> Vec<String> {
 
 pub fn options(arg_matches: &ArgMatches) -> Vec<HurlOption> {
     let mut options = vec![];
+    if let Some(value) = get::<i32>(arg_matches, "max_redirects") {
```

Comment:
The options matches where alphabetically sorted, we can keep it this way: "compressed", "insecure", "max-redirs", "negociate", "ntlm" etc...